<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppEcranDyn</class>
 <widget class="QMainWindow" name="CIhmAppEcranDyn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CIhmAppEcranDyn</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbRs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Liaison série</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>1041</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutRs">
      <item>
       <widget class="QLabel" name="laChoix">
        <property name="text">
         <string>Choix du port série : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbPorts"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbActiver">
        <property name="text">
         <string>Activer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>1041</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutTrame">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Trame :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTrame">
       <property name="text">
        <string>Aucune trame...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbEcran">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1041</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Commandes de l'écran</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>1031</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutSource">
      <item>
       <widget class="QComboBox" name="cbSource"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbSetSource">
        <property name="text">
         <string>Fixer Source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetSource">
        <property name="text">
         <string>Source ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lGetSource">
        <property name="text">
         <string>Inconnue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>1041</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutAlim">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Id : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbId"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbAlimentation">
        <property name="text">
         <string>Allumer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbEtat">
        <property name="text">
         <string>Etat ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lEtat">
        <property name="text">
         <string>???</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
